src/guide/extras/reactivity-in-depth.md

```diff
@@ -410,3 +410,93 @@ export function useMachine(options) {
 ### RxJS {#rxjs}
 
 [RxJS](https://rxjs.dev/) es una biblioteca para trabajar con flujos de eventos asíncronos. La librería [VueUse](https://vueuse.org/) proporciona el complemento [`@vueuse/rxjs`](https://vueuse.org/rxjs/readme.html) para conectar los flujos RxJS con el sistema de reactividad de Vue.
+
+## Conexión con Signals
+
+Muchos otros frameworks han introducido tipos primitivos de reactividad similares a Vue refs, bajo el término "signals":
+
+- [Solid Signals](https://www.solidjs.com/docs/latest/api#createsignal)
+- [Angular Signals](https://github.com/angular/angular/discussions/49090)
+- [Preact Signals](https://preactjs.com/guide/v10/signals/)
+- [Qwik Signals](https://qwik.builder.io/docs/components/state/#usesignal)
+
+Fundamentalmente, las signals son el mismo tipo de primitivo de reactividad que las refs de Vue. Es un contenedor de valores que proporciona seguimiento de dependencias en el acceso y activación de efectos en la mutación. En algunos contextos, las signals también están relacionadas con el modelo de renderizado donde las actualizaciones se realizan a través de suscripciones de grano fino, aunque no es un rasgo necesario para que algo se llame signal.
+
+Entre estas implementaciones, el diseño de las signals de Preact y Qwik son muy similares al [shallowRef](/api/reactivity-advanced.html#shallowref) de Vue: las tres proporcionan una interfaz mutable a través de la propiedad `.value`.
+
+### Signals de Solid
+
+El diseño de la API `useSignal()` de Solid enfatiza la segregación de lectura/escritura. Las signals se exponen como un getter de sólo lectura y un setter separado:
+
+```js
+const [count, setCount] = createSignal(0)
+
+count() // Acceder al valor
+setCount(1) // Actualizar el valor
+```
+
+Observa cómo la signal `count` pueden ser pasadas sin el setter. Esto asegura que el estado nunca puede ser mutado a menos que el setter también sea explícitamente expuesto. Si esta garantía de seguridad justifica la sintaxis más verbosa podría estar sujeto a los requisitos del proyecto y el gusto personal - pero en caso de que prefiera este estilo de API, se puede replicar fácilmente en Vue:
```

Comment:
podría pasarse a: "la signal `count` puede ser pasada sin el setter", ahora está en plural

Reply:
good catch, fixing it!